sdks/python/apache_beam/examples/snippets/snippets_test.py

```diff
@@ -176,55 +176,55 @@ def process(self, element, lower_bound, upper_bound=float('inf')):
     # [END model_pardo_side_input_dofn]
     self.assertEqual({'a', 'bb', 'ccc'}, set(small_words))
 
-  def test_pardo_with_side_outputs(self):
-    # [START model_pardo_emitting_values_on_side_outputs]
+  def test_pardo_with_tagged_outputs(self):
+    # [START model_pardo_emitting_values_on_tagged_outputs]
```

Comment:
it should pretty straightforward, though some Python code comments that aren't part of the snippets will also need to be updated in addition to the snippet tag names. I can create a PR to update these in the doc files if you want @tgroh , let me know